modules/S02_UEFI_FwHunt.sh

```diff
@@ -31,17 +31,20 @@ S02_UEFI_FwHunt() {
 
   if [[ "${UEFI_VERIFIED}" -eq 1 ]] || { [[ "${RTOS}" -eq 1 ]] && [[ "${UEFI_DETECTED}" -eq 1 ]]; }; then
     print_output "[*] Starting FwHunter UEFI firmware vulnerability detection"
-    for EXTRACTED_FILE in "${FILE_ARR_LIMITED[@]}"; do
-      if [[ ${THREADED} -eq 1 ]]; then
-        fwhunter "${EXTRACTED_FILE}" &
-        local TMP_PID="$!"
-        store_kill_pids "${TMP_PID}"
-        WAIT_PIDS_S02+=( "${TMP_PID}" )
-        max_pids_protection "${MAX_MOD_THREADS}" "${WAIT_PIDS_S02[@]}"
-      else
-        fwhunter "${EXTRACTED_FILE}"
-      fi
-    done
+    fwhunter "${FIRMWARE_PATH_BAK}"
+    if [[ $(grep -c "FwHunt rule" "${LOG_PATH_MODULE}""/fwhunt_scan_"*) -eq 0 ]]; then
```

Comment:
this grep gives you output like the following:
```
file1:0
file2:1
file3:7
```
You probably need to add them before checking them

```
grep -c ... | cut -d: -f2 | awk '{ SUM += $1} END { print SUM }' || true) -eq 0 ]] ...
```